internal/actionfacts/archive_artifact.go

```
@@ -0,0 +1,634 @@
+
+func staticArchiveArtifactPath(value string) bool {
+	if value == "" || value == "-" || value == "." || value == ".." {
+		return false
+	}
+	if strings.ContainsAny(value, "*?[]") || strings.Contains(value, "$") {
+		return false
+	}
+	return true
+}
+
+func classifyArchiveArtifactProducers(out *parseOutput, command *CommandFact) {
```

golangci-lint (2.13.2)

[error] func classifyArchiveArtifactProducers is unused

(unused)